web/src/pm_list_data.js

```diff
@@ -57,6 +58,7 @@ export function get_conversations() {
                 // always present. We may want to use this space for a
                 // bot icon in the future.
```

Comment:
We should delete this comment, which suggests the exact product change made here :)

Reply:
I pushed an update changing this comment, and also dropping setting a bogus value for the circle color here that won't be used. 

Can you check if we should do something similar in the recent conversations commit, and address the tests?

Reply:
Sure, working on it.

Reply:
@timabbott  , I have removed it from the recent conversations commit and updated the node-tests as well. Could you please review it again? Thank you :)